### PR TITLE
fix: auto-attach consumed batches through manufacturing (#329) [TEMP]

### PR DESCRIPTION
**Pull request by nmimsnikhil**
Fetched batch number in stock_entry.py and applied to item

Before:
Batch_no_Task

After:
batch_number_fix

### CODE REVIEW

**Review by Alchez — DISMISSED** [review]

**Comment by Alchez** [comment]
@bhattdevarsh, @nmimsnikhil, make this upstream to Frappe as well

File: erpnext/stock/doctype/stock_entry/stock_entry.py [hunk 1]
```diff
@@ -434,17 +434,20 @@ def set_actual_qty(self):
...
 
-	def set_serial_nos(self, work_order):
+	def set_serial_batch_nos(self, work_order):
 		previous_se = frappe.db.get_value("Stock Entry", {"work_order": work_order,
 				"purpose": "Material Transfer for Manufacture"}, "name")
 
 		for d in self.get('items'):
-			transferred_serial_no = frappe.db.get_value("Stock Entry Detail",{"parent": previous_se,
-				"item_code": d.item_code}, "serial_no")
+			transferred_serial_no, transferred_batch_no= frappe.db.get_value("Stock Entry Detail",{"parent": previous_se,
```
Comment (Alchez) [inline erpnext/stock/doctype/stock_entry/stock_entry.py]:
```suggestion
			transferred_serial_no, transferred_batch_no = frappe.db.get_value("Stock Entry Detail",{"parent": previous_se,
```

**Comment by nmimsnikhil** [comment]
ERPNext Pull Request Link:
https://github.com/frappe/erpnext/pull/20806 Hotfix
https://github.com/frappe/erpnext/pull/20808 Develop

**Comment by Alchez** [comment]
@neilLasrado @forellana-digithinkit 

Nabin added a comment upstream saying that this needs a bigger fix, but we do need an interim solution until that is made. Should we merge this one, and revert when we see changes made to this?

**Pull request merged**